package.json

```diff
@@ -19,6 +20,19 @@
   },
   "homepage": "https://github.com/HTTPArchive/custom-metrics#readme",
   "devDependencies": {
-    "fugu-api-data": "^1.15.0"
+    "fugu-api-data": "^1.15.0",
+    "jest": "^29.7.0",
+    "webpagetest": "github:max-ostapenko/webpagetest-api#post-request-support-for-runTest"
```

Comment:
@rviscomi @tunetheweb could you please fork [catchpoint/WebPageTest.api-nodejs](https://github.com/catchpoint/WebPageTest.api-nodejs) under [HTTPArchive](https://github.com/HTTPArchive) so that the dependency is controlled by the org.

Reply:
Done: https://github.com/HTTPArchive/WebPageTest.api-nodejs